build/assets/tuned/openshift-node-performance

```diff
@@ -121,12 +129,24 @@ avc_cache_threshold=8192
 initrd_remove_dir=
 initrd_dst_img=
 initrd_add_dir=
+
 # overrides cpu-partitioning cmdline
-cmdline_cpu_part=+nohz=on rcu_nocbs=${isolated_cores} tuned.non_isolcpus=${not_isolated_cpumask} intel_pstate=disable nosoftlockup
+cmdline_cpu_part=+nohz=on rcu_nocbs=${isolated_cores} tuned.non_isolcpus=${not_isolated_cpumask} systemd.cpu_affinity=${not_isolated_cores_expanded} intel_iommu=on iommu=pt
```

Comment:
Why do we not include nohz_full ?

Reply:
@MarSik IIRC it was some downsides regarding using `nohz_full`, can you please confirm?

Reply:
Added under the real time workload hint - https://github.com/openshift-kni/performance-addon-operators/pull/858/files#diff-73267c64c70b410234c47702b68c892fbbe79bef657f0658068e3de80e07b94dR143